packages/foreman/rubygem-hammer_cli/rubygem-hammer_cli.spec

```diff
@@ -4,8 +4,8 @@
 %global gem_name hammer_cli
 %global confdir hammer
 
-%global release 1
-%global prerelease .pre.develop
+%global release 2
+%global prerelease pre.develop
```

Comment:
I wonder if we should add a 
```
%global dotprerelease %{?prerelease:.}%{?prerelease}
```

Reply:
That makes the logic less repetitive so +1. Might eventually be worth a macro in tfm-build to capture it in a single place.